bin/crisper

```diff
@@ -12,27 +12,68 @@
 // jshint node: true
 'use strict';
 
-var nopt = require('nopt');
 var fs = require('fs');
 var path = require('path');
 var crisp = require('../index');
 
-var args = nopt(
+var cliArgs = require('command-line-args');
+
+var cli = cliArgs([
+  {
+    name: 'source',
+    alias: 's',
+    type: String,
+    defaultOption: true,
+    description: 'Input HTML file to split into HTML and JS files'
+  },
+  {
+    name: 'html',
+    alias: 'h',
```

Comment:
This already has to be a 2.0 release, since the behavior with no flags has changed. If we want to change it, now is a good time, but I'm not confident that it's super helpful to change it, since running ./crisper -h will still print a help message.